frontend/src/components/TransactionsPerSecond/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo } from "react";
+import { useDispatch } from "react-redux";
+import BigNumber from "bignumber.js";
+
+import { useRedux } from "hooks/useRedux";
+import {
+  ActionStatus,
+  LedgerTransactionHistoryFilterType,
+  Network,
+} from "types";
+import { SectionCard } from "components/SectionCard";
+import {
+  VerticalBarChart,
+  TimeRange,
+} from "components/charts/VerticalBarChart";
+import { fetchTransactionsHistoryAction } from "ducks/transactions";
+import "./styles.scss";
+import { AmountInfoCard } from "components/AmountInfoCard";
+import { Icon } from "@stellar/design-system";
+
+export const TransactionsPerSecond = () => {
+  const { transactions, ledgers } = useRedux("transactions", "ledgers");
+  const dispatch = useDispatch();
+  const lastLedger = ledgers.lastLedgerRecords[0];
+  const selectedTimeInterval = LedgerTransactionHistoryFilterType["30D"];
```

Comment:
I think something needs to change in types/index.ts, since the "30D" endpoint is now `month`, right? 
eg. https://github.com/stellar/dashboard/blob/08acf2a9f7397e881d73bfb45dd2a49b43b22688/backend/src/routes.ts#L33

when I tried running this I got an error that the 30D endpoint doesnt exist